cdk/lib/security-hq.ts

```diff
@@ -118,6 +120,14 @@ dpkg -i /tmp/installer.deb`,
       },
     });
 
+    // TODO reduce TTL and point to new ALB after going live.
+    const oldElb = template.getResource('LoadBalancer') as CfnLoadBalancer;
+    new GuCname(this, 'security-hq.gutools.co.uk', {
+      name: 'security-hq.gutools.co.uk',
```

Comment:
Although we don't have a CODE stack, I think it's safest to make this stage aware, otherwise if/when we try to create a CODE stack, it'll fail immediately.